types/office-js-preview/index.d.ts

```diff
@@ -325,7 +325,47 @@ declare namespace Office {
      * @param useShortNamespace True to use the shortcut alias; otherwise false to disable it. The default is true.
      */
     function useShortNamespace(useShortNamespace: boolean): void;
+    /**
+     * Represents the add-in.
```

Comment:
It is in a way add-ins runtime settings. We wanted to avoid using the work runtime. 
Using this object you can control addins display, startup behavior, etc.

Reply:
Can we add that sort of information to the description?